pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -90,7 +93,9 @@ func (q *graceTerminateRSList) flushList(handler func(rsToDelete *listItem) (boo
 		}
 		if deleted {
 			klog.Infof("lw: remote out of the list: %s", name)
-			q.remove(rs)
+			if _, ok := q.list[rs.String()]; ok {
+				delete(q.list, rs.String())
+			}
```

Comment:
```suggestion
                      // same as q.remove(rs) but without attempting to hold
                      // the lock.
			if _, ok := q.list[rs.String()]; ok {
				delete(q.list, rs.String())
			}
```

Reply:
it is ok, I was being too pedantic, just leave it as is